test/CMakeLists.txt

```diff
@@ -69,16 +69,17 @@ if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
     list(APPEND SRC graph_rewrites/enter_in_catalog_test.cc)
     list(APPEND SRC graph_rewrites/remove_ngraphassigns.cc)
     list(APPEND SRC graph_rewrites/test_replace_optimizer.cpp)
+    list(APPEND SRC graph_rewrites/test_ng_var_update_ng_tensor.cc)
 endif()
 
 if(NGRAPH_TF_USE_GRAPPLER_OPTIMIZER)
     list(APPEND SRC graph_rewrites/config_for_grappler_test.cc)
     list(APPEND SRC graph_rewrites/partial_shapes_test.cc)
 endif()
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -DNDEBUG")
```

Comment:
Please add a comment here about the DNDEBUG. Maybe the link with `mrry`'s comments